src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
+@import url(./header.scss);
+@import url(./card.scss);
+
+$card-width: 200px;
+$gap: 50px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.main {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main__container {
+  padding-top: 50px;
+  padding-bottom: 70px;
+  display: flex;
+  gap: 50px;
```

Comment:
gap 50px also applied for vertical spacing but it should be 48px 
![image](https://user-images.githubusercontent.com/54936044/227361946-17b07232-7cf7-49b4-83fd-d09d72a5f107.png)